libs/cardano-ledger-core/cardano-ledger-core.cabal

```
@@ -114,7 +114,7 @@ library
         measures,
         microlens,
         network,
-        nothunks ^>=0.1.5,
+        nothunks ^>=0.2,
```

Let's not make it too restrictive
```suggestion
        nothunks >=0.1.5 && <0.3,
```